data/bases_pt_BR.dat

```diff
@@ -1,21 +1,21 @@
 [Stolen Computer Time]
 name = Tempo Roubado de Computação
-description = (1 CPU) Invade um computador qualquer. Não posso construir nada nesta base, e ela possui apenas um único computador lento. A chance de detecção também é um pouco alta.
+description = (1 CPU) Invade um computador aleatório. Não posso construir nada nesta base, e ela possui apenas um único computador lento. A chance de detecção também é um pouco alta.
 flavor = Serviço de PC
 
 [Server Access]
 name = Acesso de Servidor
-description = (10 CPUs) Compra tempo de processador de uma empresa qualquer. Não posso construir nada nesta base, e ela possui apenas um único computador.
-flavor_list = Servidor Dedicado | Aluguel de Ponto | Hospedagem
+description = (10 CPUs) Compra tempo de processador de uma empresa entre muitas. Não posso construir nada nesta base, e ela possui apenas um único computador.
+flavor_list = Servidor Dedicado | Aluguel de Nó | Hospedagem
 
 [Datacenter]
 name = Centro de Dados
 description = (35 CPUs) Compra tempo de processador de uma empresa focada em poder de processamento bruto. Preciso de uma identidade falsa para preencher alguns formulários.
-flavor_list = Datacenter | Centro de Dados | Hospedagem Exclusiva
+flavor_list = Centro de Dados | Centro de Dados | Hospedagem Premium
```

Comment:
"Centro de Dados" twice ?

Reply:
I translated based on the [bases_en_US.dat](https://github.com/singularity/singularity/blob/master/data/bases_en_US.dat) file and in that file there are "Datacenter" and "Data Center". I didn't see the difference between them and I don't know the reason the original translator decided to translate just one of the words. Is this repetition a peculiar detail of the game?

In fact, these words could have been kept in english once in Brazil we say "Datacenter" more often than "Centro de Dados". I just followed the original translator's logic.

Reply:
i dont know either, it just appeared strange. but appart from this oddity, your translations look good :thumbsup: